<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createGameWindow</class>
 <widget class="QMainWindow" name="createGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>8</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="hostChoose"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="guestChoose"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="guestLabel">
       <property name="text">
        <string>Guest</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
